CSharpProvider/tests/test-queries.sh

```
@@ -0,0 +1,100 @@
+echo "=== Building ==="
+dotnet build "$REPO_ROOT/CSharpProvider/CSharpProvider.csproj" || exit 1
+
+# Start server
+echo "=== Starting server on port $PORT ==="
+kill $(lsof -ti :"$PORT") 2>/dev/null
+sleep 1
+dotnet run --project "$REPO_ROOT/CSharpProvider" -- --port "$PORT" > /tmp/csharp-test.log 2>&1 &
+SERVER_PID=$!
+trap "kill $SERVER_PID 2>/dev/null" EXIT
+
+for i in $(seq 1 60); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)